### PR TITLE
add(selenium-webdriver): SeleniumServer to remote.d.ts

### PR DESCRIPTION
**Pull request by ajaks328**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by cnishina** [comment]
Please add tests to `types/selenium-webdriver/test/remote.ts`

**Comment by ajaks328** [comment]
@cnishina done

File: types/selenium-webdriver/remote.d.ts [hunk 1]
```diff
@@ -158,6 +158,53 @@ export namespace DriverService {
...
+ */
+export class SeleniumServer extends DriverService {
+    /**
+     * @param {string} jar Path to the Selenium server jar.
+     * @param {SeleniumServer.Options=} opt_options Configuration options for the
+     *     server.
+     * @throws {Error} If the path to the Selenium jar is not specified or if an
+     *     invalid port is specified.
+     **/
+    constructor(jar: String, opt_options: SeleniumServer.Options);
```
Comment (andy-ms) [inline types/selenium-webdriver/remote.d.ts]: Should be `string` instead of `String`

File: types/selenium-webdriver/remote.d.ts [hunk 2]
```diff
@@ -158,6 +158,53 @@ export namespace DriverService {
...
+     * - `args` - The arguments to pass to the service. If a promise is provided,
+     *     the service will wait for it to resolve before starting.
+     * - `jvmArgs` - The arguments to pass to the JVM. If a promise is provided,
+     *     the service will wait for it to resolve before starting.
+     * - `env` - The environment variables that should be visible to the server
+     *     process. Defaults to inheriting the current process's environment.
+     * - `stdio` - IO configuration for the spawned server process. For more
+     *     information, refer to the documentation of `child_process.spawn`.
+     *
+     * @typedef {{
```
Comment (andy-ms) [inline types/selenium-webdriver/remote.d.ts]: The comment shouldn't be defining types, the actual code should.

Comment (cnishina) [reply types/selenium-webdriver/remote.d.ts]: From the name `Options`, I imagine all of these fields are optional?

```
Options {
  loopback?: boolean|undefined,
  ...
}
```

**Comment by cnishina** [comment]
Thanks for adding the test.

File: types/selenium-webdriver/remote.d.ts [hunk 3]
```diff
@@ -190,18 +190,15 @@ export namespace SeleniumServer {
...
-     *   args: !(Array<string>|IThenable<!Array<string>>),
-     *   jvmArgs: (!Array<string>|!IThenable<!Array<string>>|undefined),
-     *   env: (!Object<string, string>|undefined),
-     *   stdio: (string|!Array<string|number|!stream.Stream|null|undefined>|
-     *           undefined)
-     * }}
      */
-    interface Options { }
+    interface Options {
+        loopback?: boolean|undefined;
```
Comment (andy-ms) [inline types/selenium-webdriver/remote.d.ts]: If a value is optional there's no need to specify `|undefined`.

File: types/selenium-webdriver/remote.d.ts [hunk 4]
```diff
@@ -158,6 +158,50 @@ export namespace DriverService {
...
+     *     invalid port is specified.
+     **/
+    constructor(jar: string, opt_options?: SeleniumServer.Options);
+}
+
+export namespace SeleniumServer {
+    /**
+     * Options for the Selenium server:
+     *
+     * - `loopback` - Whether the server should only be accessed on this host's
```
Comment (andy-ms) [inline types/selenium-webdriver/remote.d.ts]: Move the comments to the actual members, as in:
```ts
/** Whether the server should only be accessed on this host's loopback address. *?
loopback?: boolean;
```

**Comment by andy-ms** [comment]
Thanks!

**Comment by cnishina** [comment]
@ajaks328 thanks for the pull request! @andy-ms thanks!

**Pull request merged**